app/code/Magento/AdvancedPricingImportExport/etc/module.xml

```diff
@@ -6,6 +6,5 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_AdvancedPricingImportExport" >
-    </module>
+    <module name="Magento_AdvancedPricingImportExport" />
```

Comment:
XML should not have space before `/>` or `>`

Reply:
Space removed